.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    # Runs a set of commands using the runners shell
+    - name: Setup ROS
+      uses: betwo/github-setup-catkin@v1.0.0
+      with:
+        # Version range or exact version of ROS version to use, using SemVer's version range syntax.
+        ros-version: # optional, default is melodic
+        # Root directory of the catkin workspace
+        workspace: ./Simulator 
+    - name: Install catkin tools
+      run: sudo apt-get install python-catkin-tools
+    - name: Compilation settings
+      run: |
+        sudo apt-get install gcc-8 g++-8
+        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 700 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+        sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+    - name: Setup AirSim 
+      run: |
+        ./setup.sh
+        ./build.sh
+      working-directory: ./AirSim 
+    - name: Build catkin ws
+      run: |
+        set -ex o pipefail
+        catkin build airsim_ros_interface
```

Comment:
why not just do `catkin build`? We probably want to also build all future packages in this workspace right?

Reply:
yeah, also less sensitive to package name changes